streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -239,10 +240,8 @@ public static String safeUniqueTestName(final Class<?> testClass, final TestName
      * Used by tests migrated to JUnit 5.
      */
     public static String safeUniqueTestName(final Class<?> testClass, final TestInfo testInfo) {
-        final String displayName = testInfo.getDisplayName();
         final String methodName = testInfo.getTestMethod().map(Method::getName).orElse("unknownMethodName");
-        final String testName = displayName.contains(methodName) ? methodName : methodName + displayName;
-        return safeUniqueTestName(testClass, testName);
+        return safeUniqueTestName(testClass, methodName + "_" + Uuid.randomUuid().toString());
```

Comment:
With the current method, we end up with a non-unique name for the parameterized tests. Therefore, I suggest to just use the name of the method with a uuid.